src/blueapi/cli/amq.py

```diff
@@ -51,12 +51,13 @@ def on_progress_event_wrapper(
                 on_progress_event(event)
 
         self.app.subscribe(
-            self.app.destinations.topic("public.worker.event"), on_event_wrapper
+            self.app.destinations.topic("public.worker.events"), on_event_wrapper
```

Comment:
We say: How come it's changed to events from event - the convention for topics/queues is singular (we believe)

Reply:
I wasn't aware of the convention, I'll change it back no problem